contracts/eth/nearprover/UPGRADE.md

```diff
@@ -26,6 +26,8 @@ To deploy the prover with the mock run: <br/>
 
 2. Upgrade the bridge address for the provided prover:<br/>
 `$ make upgrade-provers-bridge-address-to PROVER=<PROVER_ADDRESS_HERE> NEW_BRIDGE=<BRIDGE_ADDRESS_HERE>`
+or
+`$ make upgrade-provers-bridge-address-to-with-ledger PROVER=<PROVER_ADDRESS_HERE> NEW_BRIDGE=<BRIDGE_ADDRESS_HERE> LEDGER_KEY_PATH=<LEDGER_KEY_PATH_HERE>`
```

Comment:
Why in step 1 it says `eth-custodian` rather than `nearprover`?

Reply:
It is a typo and it is already fixed by @sept-en at 0b128d9c3a06aac2dc115182e4afaf36861f541d